testing/test_ASH.py

```diff
@@ -162,6 +163,10 @@ async def test_botAssignment(self):
             AntiSpamHandler(False)
 
         AntiSpamHandler(commands.Bot(command_prefix="!"))
+        AntiSpamHandler(commands.AutoShardedBot(command_prefix="!"))
+        discord.Client()
+        discord.AutoShardedClient()
```

Comment:
You're not testing anything with this line?
Did you mean `AntiSpamHandler(discord.AutoShardedClient())`

Reply:
aa yea hmm.. my bad!